Mathlib/Order/RelIso/Basic.lean

```diff
@@ -719,6 +719,14 @@ def prodLexCongr {α₁ α₂ β₁ β₂ r₁ r₂ s₁ s₂} (e₁ : @RelIso 
 def relIsoOfIsEmpty (r : α → α → Prop) (s : β → β → Prop) [IsEmpty α] [IsEmpty β] : r ≃r s :=
   ⟨Equiv.equivOfIsEmpty α β, @fun a => isEmptyElim a⟩
 
+/-- The lexicographic sum of `r` plus an empty relation is isomorphic to `r`. -/
+def sumLexEmpty (r : α → α → Prop) (s : β → β → Prop) [IsEmpty β] : Sum.Lex r s ≃r r :=
+  ⟨Equiv.sumEmpty _ _, by simp⟩
```

Comment:
This should include a composition with the equiv `A ≃ Lex A`, which I forget the name of.

Reply:
(and should have `@[simps]`)

Reply:
> the equiv `A ≃ Lex A`, which I forget the name of

`toLex`